selector/methods/distance.py

```diff
@@ -60,28 +61,39 @@ class MaxMin(SelectionBase):
     Structure‐Activity Relationships 21.6 (2002): 598-604.
     """
 
-    def __init__(self, fun_dist=None):
+    def __init__(self, fun_dist=None, ref_index=None):
         """
         Initializing class.
 
         Parameters
         ----------
         fun_distance : callable
             Function for calculating the pairwise distance between sample points.
-            `fun_dist(X) -> X_dist` takes a 2D feature array of shape (n_samples, n_features)
+            `fun_dist(x) -> x_dist` takes a 2D feature array of shape (n_samples, n_features)
             and returns a 2D distance array of shape (n_samples, n_samples).
+        ref_index: int, list, optional
+            Index of the reference sample to start the selection algorithm from.
+            It can be an integer, or a list of integers or None. When None, the medoid center is chosen as the reference
+            sample. Default is "None".
```

Comment:
@FanwangM: Remove `Default is "None"` from here and other instances below. We decided not to mention the default in the docstring, the user can see it from line 64. This makes maintaining the documentation easier, and we have been trying to follow this across packages.

Reply:
Fixed in [07f8abe](https://github.com/theochem/Selector/pull/249/commits/07f8abeed306f2b9b0c6c56a403e8b74fc68b737).